pandas/tests/io/parser/test_python_parser_only.py

```diff
@@ -488,3 +489,51 @@ def test_header_int_do_not_infer_multiindex_names_on_different_line(python_parse
     )
     expected = DataFrame({"a": ["a", "c", "f"]})
     tm.assert_frame_equal(result, expected)
+
+
+def test_no_thousand_convert_for_non_numeric_cols(python_parser_only):
+    # GH#50270
+    parser = python_parser_only
+    data = """a;b;c
+0000,7995;16,000.1;0
+3,03,001,00514;0;4,001
+4923,600,041;23,000;131
+"""
```

Comment:
just for my understanding, why change this compared with the original issue https://github.com/pandas-dev/pandas/issues/50270 ?

I think it would be good to include the original example (either in addition to or instead of this one)

Reply:
I’ve added an example similar to the original one to the `pandas/tests/io/parser/common/test_common_basic.py` early. Now I added the original example in `pandas/tests/io/parser/test_python_parser_only.py` as you suggested.

Sometimes there are many possibilities where to write a test, for example for `parser`. 
I will be grateful if you could advise me on how to choose the right place for a test.

Reply:
yeah it's confusing sometimes...I think this is fine, as you're testing the Python parser specifically